File: java_tasks/begin10.js
let a = Number(prompt("Введите первое ненулевое число"));
let b = Number(prompt("Введите второе ненулевое число"));

let Sum = (a * a) + (b * b);
let Dif = (a * a) - (b * b);
let Prd = (a * a) * (b * b);
let Qnt = (a * a) / (b * b);

alert("Сумма квадратов: " + Sum + "\n" +
    "Разность квадратов: " + Dif + "\n" +
    "Произведение квадратов: " + Prd + "\n" +
    "Частное квадратов: " + Qnt);

File: java_tasks/begin10.java
package java_tasks;

import java.util.Scanner;

class JavaTasks_Begin10 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Введите первое ненулевое число: ");
        float a = scanner.nextFloat();

        System.out.print("Введите второе ненулевое число: ");
        float b = scanner.nextFloat();

        float Sum = (a * a) + (b * b);
        float Dif = (a * a) - (b * b);
        float Prd = (a * a) * (b * b);
        float Qnt = (a * a) / (b * b);

        System.out.println("Сумма квадратов: " + Sum);
        System.out.println("Разность квадратов: " + Dif);
        System.out.println("Произведение квадратов: " + Prd);
        System.out.println("Частное квадратов: " + Qnt);
        scanner.close();
    }
}
